arm-resources/2014-04-01-preview/swagger/resources.json

```diff
@@ -1508,9 +1508,137 @@
+            },
+            "description": "The policy definition properties"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/PolicyDefinition"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "PolicyDefinitions"
+        ],
+        "operationId": "PolicyDefinitions_Get",
+        "description": "Gets policy definition.",
+        "parameters": [
+          {
+            "name": "policyDefinitionName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The policy definition name."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/PolicyDefinition"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "PolicyDefinitions"
+        ],
+        "operationId": "PolicyDefinitions_Delete",
+        "description": "Deletes policy definition.",
+        "parameters": [
+          {
+            "name": "policyDefinitionName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The policy definition name."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": ""
+          }
+        }
+      }
     }
   },
   "definitions": {
+    "PolicyDefinitionProperties": {
+      "properties": {
+        "description": {
+          "type": "string",
+          "description": "Gets or sets the policy definition description."
+        },
+        "displayName": {
+          "type": "string",
+          "description": "Gets or sets the policy definition display name."
+        },
+        "policyRule": {
+          "type": "object",
+          "description": "The policy rule json."
+        }
+      },
+      "description": "Policy definition properties."
+    },
+    "PolicyDefinition": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/PolicyDefinitionProperties",
+          "description": "Gets or sets the policy definition properties."
+        },
+        "name": {
+          "type": "string",
+          "description": "Gets or sets the policy definition name."
+        }
+      },
+        "description": "Policy definition."
+      },
```

Comment:
Do you want properties to be "flattened"? If so, apply `"x-ms-azure-resource": "true"` extension